src/cards.css

```diff
@@ -0,0 +1,82 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 166px;
+  padding: 32px 16px 16px;
+  background-color: #fff;
```

Comment:
Move all repeatable colors to CSS variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)